### PR TITLE
CSS Modules code samples

### PR DESCRIPTION
**Pull request by estelle**
The examples for the CSS module landing pages.

animations, background & borders, and blending & compositing are already live on /content/ - this will enable me to pull it out of those pages and embed code examples as iframes with {{EmbedGHLiveSample}}

### CODE REVIEW

**Review by teoli2003 — CHANGES_REQUESTED** [review]
The indentation of these files is not correct (see for example the HTML code).

Can you run them through _prettier_? (We should likely run this whole repo through it at some point)

I use the `npm prettier -w <filename>` command on the command line.

File: modules/animation.html [hunk 1]
```diff
@@ -0,0 +1,204 @@
...
+}
+ </style>
+</head>
+<body>
+<input type="checkbox" id="animate" aria-label="Toggle the play state of the animation"><!-- See aria-label: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-label -->
+<label for="animate">the animation</label>
+<div class="root">
+  <i></i><i></i><i></i><i></i><i></i><i></i><i></i><i></i><i></i><i></i><i></i><i></i><i></i><i></i><i></i><i></i><i></i><i></i><i></i><i></i><i></i><i></i><i></i><i></i><i></i><i></i><i></i><i></i><i></i><i></i><i></i><i></i><i></i><i></i><i></i><i></i><i></i><i></i><i></i><i></i><i></i><i></i><i></i><i></i><i></i><i></i><i></i><i></i><i></i><i></i><i></i><i></i><i></i>
+  <div class="cloud"></div>
+  <div class="ground"><div>
```
Comment (teoli2003) [inline modules/animation.html:201]:
```suggestion
  <div class="ground"></div>
```

File: modules/basicUI.html [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+  </style>
+</head>
+<body>
+<div><span contenteditable>Edit this text </span></div>
+<fieldset><legend>Play with these fake form controls</legend>
+    <input type="checkbox" id="check">
+    <input type="radio" name="a">
+    <input type="radio" name="a">
+    <input type="range">
+    <progress>
```
Comment (teoli2003) [inline modules/basicUI.html:45]: I think this is the right way of doing it:
```suggestion
    <progress></progress>
```

File: modules/filters.html [hunk 3]
```diff
@@ -0,0 +1,153 @@
...
+  <button type="reset">Reset</button>
+</fieldset>
+</form>
+</article>
+<script>
+const image = document.querySelector('img');
+const controls = document.querySelectorAll('input');
+let resetValues = new Array();
+
+for(control of controls) {
```
Comment (teoli2003) [inline modules/filters.html:86]: Typically, the missing space – it should be `for (` – will be fixed by Prettier.

File: modules/animation.html (hunk 1 above)
Comment (estelle) [reply modules/animation.html]: good catch. thanks

File: modules/filters.html [hunk 4]
```diff
@@ -0,0 +1,182 @@
...
+            opacity() +
+            saturate() +
+            sepia()
+        );
+      }
+      /* blur()*/
+      function blur() {
+        let blurValue = document.getElementsByName("blur")[0];
+        return "blur(" + blurValue.value + "rem) ";
+      }
```
Comment (teoli2003) [inline modules/filters.html:130]: I removed the comment as it doesn't bring anything: it is merely the function's name that appears on the following line. I'm open to another comment, though. (The others are the recommendations that came out of the Modernize JS project)
```suggestion

      function blur() {
        const blurValue = document.getElementsByName("blur")[0];
        return `blur(${blurValue.value}rem) `;
      }
```

File: modules/filters.html [hunk 5]
```diff
@@ -0,0 +1,182 @@
...
+      /* brightness()*/
+      function brightness() {
+        let brightnessValue = document.getElementsByName("brightness")[0];
+        return "brightness(" + brightnessValue.value + ") ";
+      }
```
Comment (teoli2003) [inline modules/filters.html:135]:
```suggestion

      function brightness() {
        const brightnessValue = document.getElementsByName("brightness")[0];
        return `brightness(${brightnessValue.value}) `;
      }
```

File: modules/filters.html [hunk 6]
```diff
@@ -0,0 +1,182 @@
...
+      /* contrast()*/
+      function contrast() {
+        let contrastValue = document.getElementsByName("contrast")[0];
+        return "contrast(" + contrastValue.value + ") ";
+      }
```
Comment (teoli2003) [inline modules/filters.html:140]:
```suggestion

      function contrast() {
        const contrastValue = document.getElementsByName("contrast")[0];
        return `contrast(${contrastValue.value}) `;
      }
```

File: modules/filters.html [hunk 7]
```diff
@@ -0,0 +1,182 @@
...
+      /* drop-shadow()*/
+      function dropShadow() {
+        let dropShadowValue = document.getElementsByName("dropShadow")[0];
+        return (
+          "drop-shadow(" + dropShadowValue.value +
+          "rem " + dropShadowValue.value + "rem " + "0rem orange) "
+        );
+      }
```
Comment (teoli2003) [inline modules/filters.html:148]:
```suggestion

      function dropShadow() {
        const dropShadowValue = document.getElementsByName("dropShadow")[0];
        return `drop-shadow(${dropShadowValue.value}rem ${dropShadowValue.value}rem 0rem orange) `;
      }
```

File: modules/filters.html [hunk 8]
```diff
@@ -0,0 +1,182 @@
...
+      /* grayscale()*/
+      function grayscale() {
+        let grayscaleValue = document.getElementsByName("grayscale")[0];
+        return "grayscale(" + grayscaleValue.value + ") ";
+      }
```
Comment (teoli2003) [inline modules/filters.html:153]:
```suggestion

      function grayscale() {
        const grayscaleValue = document.getElementsByName("grayscale")[0];
        return `grayscale(${grayscaleValue.value}) `;
      }
```

File: modules/filters.html [hunk 9]
```diff
@@ -0,0 +1,182 @@
...
+      /* hue-rotate()*/
+      function hueRotate() {
+        let hueRotateValue = document.getElementsByName("hueRotate")[0];
+        return "hue-rotate(" + hueRotateValue.value + "turn) ";
+      }
```
Comment (teoli2003) [inline modules/filters.html:158]:
```suggestion

      function hueRotate() {
        const hueRotateValue = document.getElementsByName("hueRotate")[0];
        return `hue-rotate(${hueRotateValue.value}turn) `;
      }
```

File: modules/filters.html [hunk 10]
```diff
@@ -0,0 +1,182 @@
...
+      /* invert()*/
+      function invert() {
+        let invertValue = document.getElementsByName("invert")[0];
+        return "invert(" + invertValue.value + ") ";
+      }
```
Comment (teoli2003) [inline modules/filters.html:163]:
```suggestion

      function invert() {
        const invertValue = document.getElementsByName("invert")[0];
        return `invert(${invertValue.value) `;
      }
```

File: modules/filters.html [hunk 11]
```diff
@@ -0,0 +1,182 @@
...
+      /* opacity()*/
+      function opacity() {
+        let opacityValue = document.getElementsByName("opacity")[0];
+        return "opacity(" + opacityValue.value + ") ";
+      }
```
Comment (teoli2003) [inline modules/filters.html:168]:
```suggestion

      function opacity() {
        const opacityValue = document.getElementsByName("opacity")[0];
        return `opacity(${opacityValue.value}) `;
      }
```

File: modules/filters.html [hunk 12]
```diff
@@ -0,0 +1,182 @@
...
+      /* saturate()*/
+      function saturate() {
+        let saturateValue = document.getElementsByName("saturate")[0];
+        return "saturate(" + saturateValue.value + ") ";
+      }
```
Comment (teoli2003) [inline modules/filters.html:173]:
```suggestion

      function saturate() {
        const saturateValue = document.getElementsByName("saturate")[0];
        return `saturate(${saturateValue.value}) `;
      }
```

File: modules/filters.html [hunk 13]
```diff
@@ -0,0 +1,182 @@
...
+      /* sepia()*/
+      function sepia() {
+        let sepiaValue = document.getElementsByName("sepia")[0];
+        return "sepia(" + sepiaValue.value + ") ";
+      }
```
Comment (teoli2003) [inline modules/filters.html:178]:
```suggestion

      function sepia() {
        const sepiaValue = document.getElementsByName("sepia")[0];
        return `sepia(${sepiaValue.value}) `;
      }
```

**Review by teoli2003 — CHANGES_REQUESTED** [review]
Thank you. I reviewed the JS, and then we are good to land this.

**Review by teoli2003 — APPROVED** [review]
Yeah!

**Pull request merged**